src/app/legacy/psammead-styles/src/__snapshots__/fonts.test.js.snap

```diff
@@ -678,7 +678,7 @@ exports[`Psammead Styles - Fonts should match REITH SERIF LIGHT base font url 1`
 "
   @font-face {
     font-family: \\"ReithSerif\\";
-    src: url(\\"https://gel.files.bbci.co.uk/r2.511/BBCReithSerif_W_Lt.woff2\\") format(\\"woff2\\"), url(\\"https://gel.files.bbci.co.uk/r2.511/BBCReithSerif_W_Lt.woff\\") format(\\"woff\\");
+    src: url(\\"https://gel.files.bbci.co.uk/r2.512/subsets/BBCReithSerif_WNumbers_Lt.woff2\\") format(\\"woff2\\"), url(\\"https://gel.files.bbci.co.uk/r2.512/subsets/BBCReithSerif_WNumbers_Lt\\") format(\\"woff\\");
```

Comment:
Is it correct that we are dropping the file extension .woff?

Reply:
should it be `...512/subsets/BBCReithSerif_WNumbers_Lt.woff\\` ?

Reply:
Nice spot!